modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/dhcp.lua

```diff
@@ -255,6 +255,22 @@ o = s:taboption("general", DynamicList, "notinterface",
 o.optional = true
 o:depends("nonwildcard", true)
 
+s = m:section(NamedSection, "odhcpd", "odhcpd", translate("odhcpd Server Settings"))
```

Comment:
Does this test for odhcpd being available?
Are there official builds without odhcpd? If so, we might want to check which of one (or both) of the dhcp-servers are available.

E.g. like this: https://github.com/openwrt/luci/pull/990/files#diff-aed351d2580a213ca424f7a09aaa8a43R15